packages/backend-deployer/src/cdk_deployer_singleton_factory.ts

```diff
@@ -43,9 +43,14 @@ export class BackendDeployerFactory {
    */
   static getInstance = (): BackendDeployer => {
     if (!BackendDeployerFactory.instance) {
+      const { PACKAGE_MANAGER_EXECUTABLE = 'npm' } = process.env;
+      const packageManager = PACKAGE_MANAGER_EXECUTABLE.startsWith('yarn')
+        ? 'yarn'
+        : PACKAGE_MANAGER_EXECUTABLE;
```

Comment:
I'm assuming this is temporary solution but I think we should:
1. Move logic that reads `process.env` to "cli like" components (backend-cli, sandbox)
2. Once we get test passing we'll need to work in backend-cli to infer package manager kind and make this information flow into deployer.

This could be as part of this PR or follow up.

Reply:
If you choose follow up. Please `// TODO` this.

Reply:
Done e485bf5de8